specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -0,0 +1,3205 @@
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "disk access resource."
+    },
+    "DiskAccessList": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DiskAccess"
+          },
+          "description": "A list of disk access resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The uri to fetch the next page of disk access resources. Call ListNext() with this to fetch the next page of disk access resources."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List disk access operation response."
+    },
+    "DiskAccessUpdate": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "Used for updating a disk access resource."
+    },
+    "PrivateLinkResource": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkResourceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "private link resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "private link resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "private link resource type"
+        }
+      },
+      "description": "A private link resource"
+    },
+    "PrivateLinkResourceProperties": {
+      "properties": {
+        "groupId": {
+          "description": "The private link resource group id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "requiredMembers": {
+          "description": "The private link resource required member names.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        },
+        "requiredZoneNames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The private link resource DNS zone name."
+        }
+      },
+      "description": "Properties of a private link resource."
+    },
+    "DiskRestorePoint": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DiskRestorePointProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "description": "Properties of disk restore point"
+    },
+    "DiskRestorePointList": {
```

Comment:
Where is this used?

Reply:
This is for LIST operation for DiskRestorePoints. Added LIST operation on line 1603 in disk.json